plugins/hls-class-plugin/src/Ide/Plugin/Class/CodeAction.hs

```diff
@@ -43,17 +43,13 @@ import           Language.LSP.Server
 addMethodPlaceholders :: PluginId -> CommandFunction IdeState AddMinimalMethodsParams
 addMethodPlaceholders _ state param@AddMinimalMethodsParams{..} = do
     caps <- getClientCapabilities
-    pluginResponse $ do
-        nfp <- getNormalizedFilePath (verTxtDocId ^. L.uri)
-        pm <- handleMaybeM "Unable to GetParsedModule"
-            $ liftIO
-            $ runAction "classplugin.addMethodPlaceholders.GetParsedModule" state
-            $ use GetParsedModule nfp
-        (hsc_dflags . hscEnv -> df) <- handleMaybeM "Unable to GhcSessionDeps"
-            $ liftIO
-            $ runAction "classplugin.addMethodPlaceholders.GhcSessionDeps" state
-            $ use GhcSessionDeps nfp
-        (old, new) <- handleMaybeM "Unable to makeEditText"
+    runExceptT $ do
+        nfp <- getNormalizedFilePathE (verTxtDocId ^. L.uri)
+        pm <- runActionE "classplugin.addMethodPlaceholders.GetParsedModule" state
```

Comment:
these weird strings that we pass in are also a bit suboptimal. not sure what to do about it, just noticing them today

Reply:
I believe they are for shake logs, but it certainly would be nice to get rid of them

Reply:
Shake logs and I think they might be used by opentelemetry to provide string representations. I don't think we should get rid of them, unless we derive the label via TH or something like that.